src/BloomBrowserUI/bookEdit/js/synphonyApi.ts

```diff
@@ -27,13 +27,13 @@ class SynphonyApi {
       lang_data.addGrapheme(this.source.letters.split(' '));
       lang_data.addWord(this.source.moreWords.split(' '));
       lang_data.LanguageSortOrder = this.source.letters.split(' ');
+    }
```

Comment:
Previously, if there were no letters, there were no stages in the decodable reader tool, even it there were stages defined in setup. It would say "Stage 0 of 0." I'm not sure what will happen if they are not using the new "Allowed Words" feature and also have no alphabet defined. I'll check that.